frontend/pages/schedule/ManageSchedulePage/ManageSchedulePage.tsx

```diff
@@ -238,9 +244,36 @@ const ManageSchedulePage = ({
     handleTeamSelect(teams[0].id);
   }
 
+  // If team_id from URL query params is not valid, we instead use a default team
+  // either the current team (if any) or all teams (for global users) or
+  // the first available team (for non-global users)
+  const getValidatedTeamId = () => {
+    if (findAvailableTeam(selectedTeamId)) {
+      return selectedTeamId;
+    }
+    if (!selectedTeamId && currentTeam) {
+      return currentTeam.id;
+    }
+    if (!selectedTeamId && !currentTeam && !isOnGlobalTeam && availableTeams) {
+      return availableTeams[0]?.id;
+    }
+    return 0;
+  };
+
+  // If team_id or currentTeam doesn't match validated id, switch to validated id
```

Comment:
Same here. Is the comment saying something the code isn't?

Reply:
I copied comments directly from ManagePoliciesPage